error-messages/user.messages.js

```diff
@@ -3,8 +3,10 @@ const USER_ALREADY_EXIST = 'USER_ALREADY_EXIST';
 const INPUT_NOT_VALID = 'INPUT_NOT_VALID';
 const WRONG_CREDENTIALS = 'WRONG_CREDENTIALS';
 const USER_NOT_AUTHORIZED = 'USER_NOT_AUTHORIZED';
+const USER_ALREADY_AUTHORIZED = ' USER_ALREADY_AUTHORIZED'
 const EMAIL_ERROR = 'EMAIL_ERROR';
 const INVALID_PERMISSIONS = 'INVALID_PERMISSIONS';
+const INVALID_INVITATIONAL_TOKEN = 'INVALID_INVITATIONAL_TOKEN';
```

Comment:
If it is used only for admin, please, rename it to the `INVALID_ADMIN_INVITATIONAL_TOKEN` to improve readability

Reply:
fixed